pyfolio/plotting.py

```diff
@@ -1344,3 +1344,63 @@ def show_worst_drawdown_periods(returns, top=5):
     drawdown_df['net drawdown in %'] = list(
         map(utils.round_two_dec_places, drawdown_df['net drawdown in %']))
     print(drawdown_df.sort('net drawdown in %', ascending=False))
+
+
+def plot_monthly_returns_timeseries(returns, ax=None, **kwargs):
+    """
+    Plots monthly returns as a timeseries.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+         - See full explanation in tears.create_full_tear_sheet.
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    **kwargs, optional
+        Passed to seaborn plotting function.
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+    """
+
+    if ax is None:
+        ax = plt.gca()
+
+    monthly_ret_table = timeseries.aggregate_returns(returns, 'monthly')
+    monthly_ret_table = monthly_ret_table.reset_index()
+    monthly_ret_table.columns = ['year', 'month', 'returns']
+
+    # Generate month-year labels for the x-axis corresponding to
+    # the returns plotted
+    date_labels = [str(i[1]) + ' - ' + str(i[0]) for i in
```

Comment:
What's wrong with the default labels of this plot? I suppose there's too many? How about just thinning those. The current approach feels a bit brittle.